src/integrations/ethereum/renderer/screens/config/ConfigScreens/ChainScreen.js

```diff
@@ -116,15 +116,20 @@ class ChainScreen extends Component {
                 }
                 changeFunction={this.validateChange.bind(this)}
               >
+                <option value="merge">Merge</option>
```

Comment:
You're adding the Merge here, but that wasn't available until Ganache v7.7.0. UI is only upgraded to 7.5.0 :/

If it's easy to include an upgrade to latest that would be sweet! v7.7.3 has all of the latest hot fixes.

Reply:
Oops - good catch!

I've upgraded to latest :)